credentialz/gnsi-credentialz.yang

```diff
@@ -48,18 +48,56 @@ module gnsi-credentialz {
         description
             "SSH server credentials freshness-related data.";
 
-        leaf active-trust-bundle-version {
+        leaf active-trusted-user-ca-keys-version {
             type version;
             description
-                "The version of the bundle of the Certificate Authority
-                certificates a.k.a. trust bundle.";
+                "The version of the Certificate Authority keys.";
         }
 
-        leaf active-trust-bundle-created-on {
+        leaf active-trusted-user-ca-keys-created-on {
             type created-on;
             description
-                "The timestamp of the moment when the bundle of the Certificate
-                Authority certificates (a.k.a. trust bundle) was created.";
+                "The timestamp of the moment when the trusted user ca keys
+                 were created.";
+        }
+
+        leaf active-host-certificate-version {
+            type version;
+            description
+                "The version of the host certificate.";
+        }
+
+        leaf active-host-certificate-created-on {
+            type created-on;
+            description
+                "The timestamp of the moment when the host certificate
+                 was created.";
+        }
+
+        leaf active-host-certificate-file-version {
```

Comment:
What is the difference between this leaf and the one in line 64?

Reply:
Yeah, I think this is a mistake. We want to _not_ mention the word file, so yanking this one out.